app/src/androidTest/java/com/sdp/movemeet/view/profile/HomeScreenActivityToUserProfileActivity.java

```diff
@@ -33,7 +33,7 @@
 
     @Test
     public void homeScreenActivityToUserProfileActivity() {
```

Comment:
If this whole test is commented, couldn't it be deleted? What is it meant to test?

Reply:
This came from JL's PR, so I didn't want to delete it if he intended to remake it later on.